packages/components/src/custom-select-control/test/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
+/**
```

Comment:
Let's add an empty line here

```suggestion

/**
```